doc/devices/aer.rst

```diff
@@ -0,0 +1,102 @@
+The Aer device
+==============
+The ``qiskit.aer`` device provided by the PennyLane-Qiskit plugin allows you to use PennyLane
+to deploy and run your quantum machine learning models on the backends and simulators provided
+by `Qiskit Aer <https://qiskit.org/aer/>`_.
+
+You can instantiate a ``'qiskit.aer'`` device for PennyLane with:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device('qiskit.aer', wires=2)
+
+This device can then be used just like other devices for the definition and evaluation of QNodes within PennyLane.
+A simple quantum function that returns the expectation value of a measurement and depends on three classical input
+parameters would look like:
+
+.. code-block:: python
+
+    @qml.qnode(dev)
+    def circuit(x, y, z):
+        qml.RZ(z, wires=[0])
+        qml.RY(y, wires=[0])
+        qml.RX(x, wires=[0])
+        qml.CNOT(wires=[0, 1])
+        return qml.expval(qml.PauliZ(wires=1))
+
+You can then execute the circuit like any other function to get the quantum mechanical expectation value.
+
+.. code-block:: python
+
+    circuit(0.2, 0.1, 0.3)
+
+Backends
+~~~~~~~~
+
+
+Qiskit's Aer layer has several backends, for example ``'qasm_simulator'``,
+``'statevector_simulator'``, ``'unitary_simulator'``. For more information on backends, please visit the
+`Aer provider documentation <https://qiskit.org/documentation/apidoc/aer_provider.html>`_ and the
+`Qiskit Aer tutorials <https://qiskit.org/documentation/tutorials/advanced/aer/1_aer_provider.html>`_.
+
+You can change an ``'qiskit.aer'`` device's backend with
+
+.. code-block:: python
+
+    dev = qml.device('qiskit.aer', wires=2, backend='unitary_simulator')
+
+To get a current overview what backends are available you can query
+
+.. code-block:: python
+
+    dev.capabilities()['backend']
```

Comment:
This is really nice, I had no idea!

Reply:
I thought you wrote it into the previous docs ;)

Reply:
I probably did lol